### DISCUSSION TITLE
Users/Accounts Table Name: Users vs Profiles

### DISCUSSION THREAD

**Discussion by jtkeyva** [supabase/supabase#4469] [Questions]
Hello, I remember reading in a Hasura forum that the User and/or Users table is reserved in Supabase/Postgress.

I noticed in the SQL examples the the User Management Starter uses "profiles" whereas the Slack Clone uses "users".

Where is the Supabase Users info stored?

Also, a lot of people prefer singular such as "user" or "profile" but I believe User is reserved.

Which is the recommended way for Supabase?

Thanks!

**Comment by b2m9** [comment]
`auth.users` is a default table that is not accessible to the public (and this is where Supabase stores all auth data like email, provider, uuid, etc.). However, you can create a table in the `public` schema named `public.users` - which would be then accessible via the API on the client as `users`. Neither `user` nor `users` is reserved in the `public` schema.

I don't think there's a recommended way, but feel free to correct me here. I usually default to using both, `users` and `profiles` in most applications with `users` being a private table only for the users themselves (account settings, subscription info, what not), while `profiles` contains all public information like username or profile picture. But I think every team has a different guideline here.
